api/EXT_color_buffer_half_float.json

```diff
@@ -14,19 +14,19 @@
             "version_added": "17"
           },
           "firefox": {
-            "version_added": "30"
+            "version_added": "47"
```

Comment:
Firefox 30 or 39 seems to be correct here:
https://bugzilla.mozilla.org/show_bug.cgi?id=843666
https://bugzilla.mozilla.org/show_bug.cgi?id=1130616

Reply:
It appears that this extension was supported in macOS since Firefox 36, and then Windows in 47.

Reply:
Oh my, so much OS-dependence. I'll bet some of this also depends on the graphics card, but how to test for that I don't know. I know that for Chromium there are a bunch of different real machines running WebGL tests, but let's not go there unless we discover such a dependency...